specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -481,15 +528,15 @@
       "description": "Operations List response which contains list of available APIs.",
       "type": "object",
       "properties": {
-        "Value": {
+        "value": {
           "description": "List of available operationss.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ClientDiscoveryValueForSingleApi"
           }
         },
-        "NextLink": {
-          "description": "NextLink of the Response.",
+        "nextLink": {
+          "description": "Link to the next chunk of Response.",

```

Comment:
This is awkward or wrong grammatically. Maybe, "Link to the next chunk of the response" ?

Reply:
Okay. :) Thanks